packages/driver/cypress/integration/commands/net_stubbing_spec.ts

```diff
@@ -11,6 +11,13 @@ const testFail = (cb, expectedDocsUrl = 'https://on.cypress.io/intercept') => {
   })
 }
 
+let routeCount = 0
+const uniqueRoute = (route) => {
```

Comment:
probably worth commenting

```suggestion

// This function returns a globally-unique URL that can be used in an individual `it` to avoid failed assertions due to request retries leaking between `it`s.
// https://github.com/cypress-io/cypress/issues/20397
const uniqueRoute = (route) => {
```